### PR TITLE
Incursion and Mission Ledger support

### PR DESCRIPTION
**Pull request by dysath**
- Refactored the Monthly data fetch routines.
- Fixed Styling
- Fixed moar styling...

### CODE REVIEW

File: src/Repositories/Corporation/Ledger.php [hunk 1]
```diff
@@ -114,23 +145,99 @@ public function getCorporationLedgerPITotalsByMonth(int $corporation_id,
...
+    public function getCorporationLedgerTotalsByMonth(int $corporation_id,
+                                                        int $year = null,
+                                                        int $month = null,
+                                                        int $oid = 2,
+                                                        array $ref_types,
+                                                        array $add_where): Collection
+    {
+
+        $select = 'MONTH(date) as month, YEAR(date) as year, ROUND(SUM(amount)) as total, ';
+        $select .= 'ownerName' . $oid . ', ownerID' . $oid;
```
Comment (leonjza) [inline src/Repositories/Corporation/Ledger.php]: This is something we need to use the query build for.

**Comment by leonjza** [comment]
Thanks for the PR! 🎉 

I think this part of the pull is going to need some work to get the queries using the query builder  entirely. The raw string concatenation is something that I would like to completely avoid.

**Comment by dysath** [comment]
I think I see what you're saying.  Yeah, that was a bad decision on my part.  I'll put something better together this week.

**Comment by leonjza** [comment]
Thanks! 👏

**Pull request closed without merging**